### DISCUSSION TITLE
Vidstack Player Beta (OLD)

### DISCUSSION THREAD

**Discussion by mihar-22** [vidstack/player#626] [Releases]
👋 

Welcome to the Vidstack Player Beta 🎉  

- We encourage you to start from our [Quickstart](https://www.vidstack.io/docs/player/getting-started/quickstart/) docs and see where it takes you 🗺️  💻 
- [GitHub Discussions](https://github.com/vidstack/player/discussions) is now open! Feel free to open up discussions for feedback, ideas, and questions.

## What To Expect?

**Beta is not a destination but rather a quick pit stop to let all of you join in on the journey of building and using the player. We'll still be actively building from here. We strongly recommend you only go ahead with trying the Beta out if you're okay with the following:**

- Expect things not to work. We haven't had a chance to test this out across every bundler and framework. We're about ~80% confident things should be fine 😅  If you find a problem, open an issue, and we'll try to resolve it as soon as possible.
- Expect the docs to fail you. Writing good docs is hard. We've tried our best, but we _know_ that it's not good enough as things that might be obvious to us might not be to you. We'd love your help in making it better.
- Expect no out-of-the-box player skins. We're working on it! You'll be building your own skins for now, which is one of the advantages of using this library.
- Expect no YouTube/Vimeo. We know it's important to some of you, and we're working on it! 
- Expect core player features such as captions, playback rate, playback quality, picture-in-picture to be missing. We're working on it!
- The 'Core Concepts'  docs are missing, but we don't think it should affect anyone. It's more of a deep dive for advanced use cases. 

Other than that, expect to have some fun! There are some innovative ideas in this library, and it might take some getting used to, but we believe it'll be worth your time.

## Next Steps

There's so much to do....

- We're considering bringing our packages and site all into this repo (i.e., monorepo).
- We'll be working on media store integrations for React, Svelte, and Vue. The store will be used for conveniently creating two-way binding to media state.
- We're working towards the 1.0 release, and you can track our progress on the [1.0 project board](https://github.com/vidstack/player/projects/1).
- We've been designing a developer-friendly CMS for video! It's coming together nicely, and we're planning on integrating with our first stream provider (and our favourite)... **Mux!**. Lot's of plans here, and the word CMS doesn't do it justice, but that's where it'll start.
- Early discussions on how we can enable developers to build and distribute skins over a simple URL or NPM package. This will tie directly in to our dashboard.

Enjoy 🍎

**Comment by mihar-22** [comment]
**Quick personal news for anyone interested...**

My time has naturally come to an end at Reddit. I'm super grateful for the year I spent there and the first-hand experience I got with building at scale. This means I'm full-time now on Vidstack, which is both exciting and scary. We're working towards our first paid product and hoping that keep us afloat. I'll also start looking for funding and applying for the next YC batch. I'm excited to spend more time on my passion for OSS and building dev tools but slightly worried about survival (~6-8 months in the bank). Regardless, I'm equally optimistic about everything that will come -- wishing myself and everyone here some extra hard work and luck to follow.

**Reply by keytrap-x86** [reply]
Good luck, this looks promising !
